influxdb/influxdb.go

```diff
@@ -50,100 +52,84 @@ const (
 // prefix in metric namespace
 var prefix = []string{nsVendor, nsClass}
 
-// InfluxdbCollector holds data retrieved from influxDB system monitoring
-type InfluxdbCollector struct {
-	data        map[string]datum
-	service     monitor.Monitoring
-	initialized bool
+type getResponse func(url string) ([]byte, error)
+
+// influxdbCollector holds data retrieved from influxDB system monitoring
+type influxdbCollector struct {
+	urlStatistic  *url.URL
+	urlDiagnostic *url.URL
+	getResponse
 }
 
-type datum struct {
-	value interface{}
-	tags  map[string]string
+type endpoint interface {
```

Comment:
The endpoint interface isn't used anywhere in code or unit tests, so it can be removed.